.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
```

Comment:
Follows the example from https://semantic-release.gitbook.io/semantic-release/recipes/ci-configurations/github-actions#.github-workflows-release.yml-configuration-for-node-projects